api/v1alpha1/virtualmachineimage_conversion.go

```diff
@@ -405,7 +406,14 @@ func (dst *VirtualMachineImage) ConvertFrom(srcRaw conversion.Hub) error {
 		dst.Spec.ProviderRef.Namespace = src.Namespace
 	}
 
-	return nil
+	// Manually restore data.
+	restored := &VirtualMachineImage{}
+	if ok, err := utilconversion.UnmarshalData(src, restored); err != nil || !ok {
+		return err
+	}
+	dst.Status.ContentLibraryRef = restored.Status.ContentLibraryRef
+
+	return utilconversion.MarshalData(src, dst)
```

Comment:
This should be balanced by an `UnmarshalData` in the ConvertFrom? This is confusing b/c we're overloading the conversion anntotation?

Why is the ClusterVirtualMachineImage _not_ have this? Why would the scope matter here?

Reply:
This was unintentional, and should not have been in the changeset in the first place. The logic works without it as well.